bin/tritond

```diff
@@ -0,0 +1,313 @@
+
+    # We just log straight to stdout. Generally tritond is run by some process
+    # that handles collecting logging for you, like upstart or supervisord. It
+    # would be easy enough to add an option if it was needed by someone though.
+    logging.basicConfig(level=level, format=log_format, stream=sys.stdout)
+
+
+class LogFileStream(object):
+
+    def __init__(self, log_path, name, rotate_hours=12):
+        self.name = name
+        self.log_path = log_path
+        self.rotate_hours = rotate_hours
+
+        self.last_write = None
+        self.last_flush = None
+        self.last_poll = None
+        self.stream = None
+        self.stream_filename = None
+
+    def poll(self):
+        if (
+            self.last_poll
+            and time.time() - self.last_poll < FILE_POLL_INTERVAL
+        ):
+            return True
+
+        log.debug("Doing poll for file %s", self.name)
+
+        self.last_poll = time.time()
+
+        if self.stream:
+            if time.time() - self.last_write > FILE_IDLE_TIMEOUT:
+                self.close()
+                return False
+            elif time.time() - self.last_write > FILE_POLL_INTERVAL:
+                self.stream.flush()
+
+        if self.stream and not os.path.exists(self.stream_filename):
+            self.close()
+            return False
+
+        if self.stream and self.log_file_path() != self.stream_filename:
+            self.close()
+            return False
+
+        return True
+
+    def log_file_path(self):
+        now = datetime.datetime.utcnow()
+        if self.rotate_hours:
+            hour = now.hour - (now.hour % self.rotate_hours)
+            now = now.replace(
+                hour=hour,
+                minute=0,
+                second=0,
+                microsecond=0)
+        date_name_str = now.strftime('%Y%m%d%H')
+        log_file_name = "{name}-{date_name}.log".format(
+            name=self.name,
+            date_name=date_name_str)
+
+        return os.path.join(self.log_path, log_file_name)
+
+    def open(self):
+        self.stream_filename = self.log_file_path()
+        try:
+            os.makedirs(os.path.dirname(self.stream_filename))
+        except OSError:
+            pass
+
+        log.info("Opening file stream for %s: %s", self.name,
+                 self.stream_filename)
+        self.stream = open(self.stream_filename, "a")
+
+    def write(self, data):
+        self.last_write = time.time()
+        if not self.stream:
+            self.open()
+
+        self.stream.write(data)
+
+    def write_many(self, data_list):
+        self.last_write = time.time()
+        if not self.stream:
+            self.open()
+
+        self.stream.writelines(data_list)
+
+    def close(self):
+        if self.stream:
+            log.info("Closing stream for file %s", self.name)
+            self.stream.close()
+            self.stream = None
+            self.stream_filename = None
+
+
+def get_triton_config():
```

Comment:
This does call `triton.config.load_config`, this wrapper function is just to get the config path.